src/azul/service/responseobjects/storage_service.py

```diff
@@ -1,34 +1,34 @@
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from dataclasses import dataclass
+from functools import lru_cache
 from logging import getLogger
 from typing import Optional
 import boto3
 from azul import config
 
 logger = getLogger(__name__)
+AWS_S3_DEFAULT_MINIMUM_PART_SIZE = 5242880  # 5 MB; see https://docs.aws.amazon.com/AmazonS3/latest/dev/qfacts.html
```

Comment:
I don't think we need to make this configurable, btw.

Reply:
Ok then.